docs/examples/add-to-your-application/ruby/data_filtering.rb

```diff
@@ -0,0 +1,93 @@
+require 'oso'
+require 'sequel'
+require 'sinatra'
+
+require_relative './models'
+
+DB = Sequel.sqlite
+
+DB.create_table :repositories do
+  String :name
+  TrueClass :is_public
+end
+
+repositories = DB[:repositories]
+
+module DF
+class Repository < Sequel::Model(:repositories)
```

Comment:
```suggestion
  class Repository < Sequel::Model(:repositories)
```

Reply:
I have this not indented intentionally because the literalInclude will include the indents I'm pretty sure (which I don't want).